### PR TITLE
porting the donate cta modal

### PR DESCRIPTION
**Pull request by Pomax**
This has the strings located in the en-US locale file, and the modal styled as per luxpertise.

### CODE REVIEW

File: src/pages/home.js [hunk 1]
```diff
@@ -19,17 +22,81 @@ var Impact = React.createClass({
...
+    if (this.state.showModal) {
+      return this.generateDonationModal();
+    }
+    return null;
+  },
+  generateDonationModal: function() {
+    return (
+      <Modal onClose={() => this.closeModal()}>
+        <section className="donate-container">
+          <h2>We all love the web.<br/> Join Mozilla in defending it.</h2>
```
Comment (TheoChevalier) [inline src/pages/home.js]: This website is localized in `de, es, fr, it, pl`, do you mind extracting the strings here? https://github.com/mozilla/copyright/blob/master/locales/en-US/copyright.properties#L36

Comment (Pomax) [reply src/pages/home.js]: ping @gthomas to make sure this is indeed the text we want.

**Comment by flukeout** [comment]
Styling this now and also wondering. Should we set the 'dismissed' flag to true also when someone hits the Donate button? Would make sense.

**Comment by Pomax** [comment]
Hm, probably opens up too many edge cases that require intercepting the `<a>` event, so I'd say let's keep it as is, and only record dismissal on actual dismissal.

**Comment by flukeout** [comment]
👍 PR incoming against your branch shortly.

**Comment by Pomax** [comment]
@ScottDowne @alanmoo is this still lated for landing or has this been passed by?

**Comment by alanmoo** [comment]
Is the code ready? Any reason we shouldn't land it?

**Comment by ScottDowne** [comment]
I think it just needs a proper review. I don't think I got around to running it until it fell off my radar.

I think what it's adding is still valid.

**Comment by Pomax** [comment]
closing due to how long it's been.

**Pull request closed without merging**